httpbin/cmd/cmd_test.go

```diff
@@ -431,6 +465,28 @@ func TestLoadConfig(t *testing.T) {
 				MaxBodySize:            httpbin.DefaultMaxBodySize,
 				MaxDuration:            httpbin.DefaultMaxDuration,
 				AllowedRedirectDomains: []string{"foo", "bar", "baz"},
+				LogFormat:              defaultLogFormat,
+			},
+		},
+		"ok use json log format": {
+			args: []string{"-log-format", "json"},
+			env:  map[string]string{"LOG_FORMAT": "json"},
```

Comment:
Just noticed it, but this should be two separate tests, one for the CLI arg and one for the env var.  It would have caught the bug in processing the configuration I pointed out in an earlier review (where the LOG_FORMAT env var was ignored)!

Reply:
agreed will add separate test for that

Reply:
I'd be in favor of dropping this original test now, I don't think we get any real value of testing what happens when we configure the same logging via both env var and CLI flag

Reply:
Yeah true, I dropped that. Also added one more case to test if nothing provided is it using default log format or not, I don't know if it makes sense for you